twpca/model.py

```diff
@@ -248,11 +252,12 @@ def assign_factors(self):
         # done initializing factors
         return self._sess.run(assignment_ops)
 
-    def assign_warps(self, warps):
+    def assign_warps(self, warps, normalize_warps=False):
```

Comment:
Oh I think I got rid of this option in https://github.com/ganguli-lab/twpca/pull/18

We should merge that first and make sure there aren't conflicts